Compilation error on arm64

When I try to compile tktable under arm64, I get the following error:
```
./generic/tkTableTag.c: In function ‘TableResetTag’:
./generic/tkTableTag.c:215:47: error: incompatible type for argument 2 of ‘tclStubsPtr->tcl_PanicVA’
  215 |       Tcl_PanicVA("bad mojo in TableResetTag",NULL);
      |                                               ^~~~
      |                                               |
      |                                               void *
./generic/tkTableTag.c:215:47: note: expected ‘va_list’ but argument is of type ‘void *’
```
This error is repeated seven times; full build log is [here](https://buildd.debian.org/status/fetch.php?pkg=tk-table&arch=arm64&ver=2.10.5-1&stamp=1605254491&raw=0). The same happens on some other platforms: arm 32 bit soft float, alpha, sh4. General build logs for this version [here](https://buildd.debian.org/status/logs.php?pkg=tk-table&ver=2.10.5-1&suite=sid).
Tcl version is 8.6.10.

--- a/generic/tkTableTag.c
+++ b/generic/tkTableTag.c
@@ -212,7 +212,7 @@ TableResetTag(Table *tablePtr, TableTag *tagPtr)
     TableJoinTag *jtagPtr = (TableJoinTag *) tagPtr;
 
     if (jtagPtr->magic != 0x99ABCDEF) {
-      Tcl_PanicVA("bad mojo in TableResetTag",NULL);
+      Tcl_Panic("bad mojo in TableResetTag");
     }
 
     memset((VOID *) jtagPtr, 0, sizeof(TableJoinTag));
@@ -269,7 +269,7 @@ TableMergeTag(Table *tablePtr, TableTag *baseTag, TableTag *addTag)
     unsigned int prio;
 
     if (jtagPtr->magic != 0x99ABCDEF) {
-      Tcl_PanicVA("bad mojo in TableMergeTag",NULL);
+      Tcl_Panic("bad mojo in TableMergeTag");
     }
 
 #ifndef NO_TAG_PRIORITIES
@@ -432,7 +432,7 @@ TableGetTagBorders(TableTag *tagPtr,
 	    if (bottom)	{ *bottom	= tagPtr->bd[3]; }
 	    break;
 	default:
-	    Tcl_PanicVA("invalid border value '%d'\n", tagPtr->borders);
+	    Tcl_Panic("invalid border value '%d'\n", tagPtr->borders);
 	    break;
     }
     return tagPtr->borders;

--- a/generic/tkTableUtil.c
+++ b/generic/tkTableUtil.c
@@ -105,7 +105,7 @@ TableOptionBdSet(clientData, interp, tkwin, value, widgRec, offset)
 	bordersPtr	= &(tagPtr->borders);
 	bdPtr		= tagPtr->bd;
     } else {
-      Tcl_PanicVA("invalid type given to TableOptionBdSet\n",NULL);
+      Tcl_Panic("invalid type given to TableOptionBdSet\n");
 	return TCL_ERROR; /* lint */
     }
 
@@ -188,7 +188,7 @@ TableOptionBdGet(clientData, tkwin, widgRec, offset, freeProcPtr)
     } else if (type == BD_TABLE_WIN) {
 	return ((TableEmbWindow *) widgRec)->borderStr;
     } else {
-      Tcl_PanicVA("invalid type given to TableOptionBdSet\n",NULL);
+      Tcl_Panic("invalid type given to TableOptionBdSet\n");
 	return NULL; /* lint */
     }
 }

